clang/lib/CodeGen/CGStmt.cpp

```diff
@@ -628,7 +628,7 @@ CodeGenFunction::getJumpDestForLabel(const LabelDecl *D) {
 
   // Create, but don't insert, the new block.
   Dest = JumpDest(createBasicBlock(D->getName()),
-                  EHScopeStack::stable_iterator::invalid(),
+                  EHScopeStack::stable_iterator::invalid(), 0,
```

Comment:
"0" seems wrong; goto out of a StmtExpr is legal, the same way "break" is legal.  But I guess we can't actually compute the correct number here; maybe we need to do something with BranchFixups?

Reply:
(I'm okay with implementing a fix with restricted scope if you want to get something into the LLVM 18 branch.  But before we decide that, I'd like some idea of what's required to actually implement this completely.)